mscore/menus.cpp

```diff
@@ -1466,9 +1466,15 @@ PalettePanel* MuseScore::newLinesPalettePanel()
 
       TextLine* textLine = new TextLine(gscore);
       textLine->setLen(w);
-      textLine->setBeginText("VII");
+      textLine->setBeginText("Staff");
       textLine->setEndHookType(HookType::HOOK_90);
-      sp->append(textLine, QT_TRANSLATE_NOOP("Palette", "Text line"));
+      sp->append(textLine, QT_TRANSLATE_NOOP("Palette", "Staff Text line"));
```

Comment:
Not sure how to name it then though

Reply:
As requested, this PR includes the same textline twice on the palette, once as a "Staff Text Line" (which was the original Text Line) and a new "System Text Line" which is a few lines lower. Leaving the original name unchanged could cause confusion.